ci-operator/config/rh-ecosystem-edge/kernel-module-management/rh-ecosystem-edge-kernel-module-management-main.yaml

```diff
@@ -1,9 +1,31 @@
+base_images:
+  ocp_builder_rhel-8-golang-1.18-openshift-4.12:
+    name: builder
+    namespace: ocp
+    tag: rhel-8-golang-1.18-openshift-4.12
+  ubi9:
+    name: ubi
+    namespace: ocp
+    tag: '9'
 binary_build_commands: make manager
 build_root:
   image_stream_tag:
     name: release
     namespace: openshift
     tag: golang-1.18
+images:
+- dockerfile_literal: |
+    FROM ubi9
+    RUN dnf install -y https://github.com/golangci/golangci-lint/releases/download/v1.48.0/golangci-lint-1.48.0-linux-amd64.rpm
+  to: linter
```

Comment:
Looks like there is already an available image that can be use for that. It needs to be added to the `base_images` section though.

See [doc](https://docs.ci.openshift.org/docs/getting-started/examples/#how-do-i-write-a-simple-execute-this-command-in-a-container-test) for that.